ansible/roles/sonarqube/defaults/main.yml

```diff
@@ -5,24 +5,28 @@
 sonar_temporary_packages:
   - unzip
 
+# Temporary packages for pip (removed after install)
+pip_temporary_packages:
+  - boto3
+
 # Packages required on the SonarQube server (persistent)
 sonar_required_packages:
   - fontconfig
   - freetype
 
 # Install requirements
-sonar_max_map_count: 524288
-sonar_file_max: 131072
+sonar_max_map_count: "524288"
+sonar_file_max: "131072"
 sonar_limitnofile: 131072
 sonar_limitnproc: 8192
 
 # SonarQube version to install
 sonar_version: 9.7.1.62043
 
 # SonarQube URLs
-sonar_source_url: "https://binaries.sonarsource.com"
-sonar_archive: "sonarqube-{{ sonar_version }}.zip"
-sonar_download_url: "{{ sonar_source_url }}/Distribution/sonarqube/{{ sonar_archive }}"
+sonar_source_url: ""
```

Comment:
Should this be empty?

Reply:
I've just seen the subsequent update. It might be worth renaming this to `resource_bucket` or similar to avoid confusion

Reply:
This is getting populated via the pipeline as I wanted to avoid putting in the full S3 url.